cmd/tellor/main.go

```diff
@@ -156,7 +156,8 @@ func App() *cli.Cli {
 	app.Command("stake", "staking operations", stakeCmd(logSetup, logLevel))
 	app.Command("transfer", "send TRB to address", moveCmd(ops.Transfer, logSetup, logLevel))
 	app.Command("approve", "approve TRB to address", moveCmd(ops.Approve, logSetup, logLevel))
-	app.Command("balance", "check balance of address", balanceCmd)
+	//Using values from context, until we have a function that setups the client and returns as values, not as part of the context
```

Comment:
```suggestion
	// Using values from context, until we have a function that setups the client and returns as values, not as part of the context.
```
Golang commentary guidelines.
https://golang.org/doc/effective_go.html#commentary